packages/charts/examples/components/demo/advanced-usage/legend/legend-metadata/legend-metadata-example.component.ts

```diff
@@ -0,0 +1,156 @@
+import { Component, OnInit } from "@angular/core";
+import {
+    areaGrid,
+    AreaRenderer,
+    Chart,
+    ChartAssist,
+    IAccessors,
+    IAreaAccessors,
+    IChartAssistSeries,
+    IChartSeries,
+    ILineAccessors,
+    IXYScales,
+    LineAccessors,
+    LinearScale,
+    LineRenderer,
+    stackedArea,
+    stackedAreaAccessors,
+    TimeScale, XYAccessors, XYRenderer,
+} from "@nova-ui/charts";
+import moment from "moment/moment";
+
+@Component({
+    selector: "area-chart-stack-example",
+    templateUrl: "./legend-metadata-example.component.html",
+})
+export class LegendMetadataExampleComponent implements OnInit {
+    public chart: Chart;
+    public chartAssist: ChartAssist;
+    public avgSeries: IChartAssistSeries<ILineAccessors>
+
+    public ngOnInit() {
+        // areaGrid returns an XYGrid configured for displaying an area chart's axes and other grid elements.
```

Comment:
I think all the comments about area chart usage can be removed since we're focusing on the legend here.

Reply:
I think it doesn't hurt so I would like to keep them if that's fine.